### DISCUSSION TITLE
With PAM login, does this also store files as the authorized user on the file system and use their permissions to read files?

### DISCUSSION THREAD

**Discussion by dblsaiko** [mar10/wsgidav#317] [Q&A]
I'm looking for a WebDAV server that acts as the authorized user instead of accessing everything via the user the service is running as, because this will be coexisting with NFS. Even better if this also works with the trusted_auth_header option set or some kind of certificate based login. Can this do that?

Thanks!

**Comment by mar10** [comment]
No, this is not implemented right now.

We probably could add something with [os.setuid()](https://docs.python.org/3/library/os.html#os.setuid), but this would be Unix only.
See also [this post](https://stackoverflow.com/a/51502683).
It is not on the roadmap, but I am open to pull requests.

**Comment by dblsaiko** [comment]
Alright, thanks for the answer regardless!
